Makefile

```
@@ -377,4 +377,8 @@ fuzz-test-command:
 
 fuzz-test-data:
 # not joining the edgex-network due to swagger file url pointing to localhost for fuzz testing in the container
-	docker run --net host --rm -v "$$PWD/fuzz_test/fuzz_results:/fuzz_results" fuzz-edgex-go:latest core-data /restler-fuzzer/openapi/core-data.yaml
\ No newline at end of file
+	docker run --net host --rm -v "$$PWD/fuzz_test/fuzz_results:/fuzz_results" fuzz-edgex-go:latest core-data /restler-fuzzer/openapi/core-data.yaml
+
+fuzz-test-support-notifications:
```

use short name like was done with core data and core command
```suggestion
fuzz-test-notifications:
```